examples/lsc/mag240m/gnn.py

```diff
@@ -12,8 +12,12 @@
 import torch.nn.functional as F
 from torch.nn import ModuleList, Sequential, Linear, BatchNorm1d, ReLU, Dropout
 from torch.optim.lr_scheduler import StepLR
-
-from pytorch_lightning.metrics import Accuracy
+import pytorch_lightning
+if int(pytorch_lightning.__version__.split('.')[0]) < 2:
```

Comment:
Can we just create a variable `WITH_LIGHTNING_V2` to reduce the amount of `split` logic here?